src/jsMain/kotlin/team/karakum/showcase/material/utils/ShowcaseBox.kt

```diff
@@ -0,0 +1,21 @@
+package team.karakum.showcase.material.utils
+
+import mui.material.Box
+import mui.system.sx
+import react.FC
+import react.PropsWithChildren
+import web.cssom.Display
+import web.cssom.JustifyContent
+import web.cssom.em
+
+val ShowcaseBox = FC<PropsWithChildren> { props ->
```

Comment:
`Container`?
Btw should it be applied for every showcase?

Reply:
I guess not every showcase is suitable, but I can look for common places and props

Reply:
it should not be applied for every showcase though, as it breaks some of the stories (because of the default display flex)